apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/style-tour.scss

```diff
@@ -216,13 +216,27 @@ $welcome-tour-card-media-extra-padding: 14%; // temporary value, to match the pa
 	width: 100%;
 }
 
-.wpcom-editor-welcome-tour.is-mobile {
-	.wpcom-editor-welcome-tour__step {
-		&.is-with-extra-padding {
-			.components-card__media img {
-				left: $welcome-tour-card-media-extra-padding;
-				top: $welcome-tour-card-media-extra-padding;
-				width: 100% - $welcome-tour-card-media-extra-padding;
+.wpcom-editor-welcome-tour {
+	&.is-mobile {
+		.wpcom-editor-welcome-tour__step {
+			&.is-with-extra-padding {
+				.components-card__media img {
+					left: $welcome-tour-card-media-extra-padding;
+					top: $welcome-tour-card-media-extra-padding;
+					width: 100% - $welcome-tour-card-media-extra-padding;
+				}
+			}
+		}
+	}
+
+	&.is-desktop {
+		.wpcom-editor-welcome-tour__step {
+			&.is-with-extra-padding-desktop {
+				.components-card__media img {
+					left: $welcome-tour-card-media-extra-padding;
+					top: $welcome-tour-card-media-extra-padding;
+					width: 100% - $welcome-tour-card-media-extra-padding;
+				}
 			}
 		}
 	}
```

Comment:
Experimenting with a working version.
The problem here is that the new slide is desktop only, and it needs extra padding. Until now, all the images that needed extra padding were only the mobile ones, so the CSS worked.

Random thoughts:
- it'd be difficult for the consumer to apply the right CSS when there are differences between mobile and desktop classes, like in this case. 
- could we consider something like:
```js
options: {
		className: {
			desktop: 'is-with-extra-padding',
			mobile: '',
		},
	},
```

This is just a random thought, I should study it a little more, but wanted to know your idea @chriskmnds 

In that case, we could rethink the CSS as:
```css
.wpcom-editor-welcome-tour {
	.wpcom-editor-welcome-tour__step {
		&.is-with-extra-padding {
			.components-card__media img {
				left: $welcome-tour-card-media-extra-padding;
				top: $welcome-tour-card-media-extra-padding;
				width: 100% - $welcome-tour-card-media-extra-padding;
			}
		}
	}
}
```

Reply:
I don't think that's necessary. We provide `is-mobile` & `is-desktop` modifiers on the parent (`wpcom-editor-welcome-tour` in this case), so we should be able to work with those?

Reply:
p.s. happy to add those modifiers to the steps as well if it will ease with usage. But will it? 🤔

Reply:
My concert was about creating another class `is-with-extra-padding-desktop`, with the same content, as in the example above!
If we simply use `is-with-extra-padding`, I have to tell to apply that class only on mobile in some slides and only on desktop in others 🤔  

Well anyway, a simple CSS solution is the one above, it seemed a little confusing tho 🤔

Reply:
Up to you on this one! I'd be fine with either way I think. But if we'll have no use for the modifiers added by Tour Kit, we should probably remove them. 

p.s. whatever we choose, let's also communicate in the README the distinction between mobile & desktop at the places where it's mentioned (cause so far it's only deep inside Tour Kit). I meant to add a note about the `is-mobile` and `is-desktop` modifiers, so may as well take care of it now. :-)

Reply:
> I'd be fine with either way I think.

but happy to rethink a little more. I'm fiddling with this structure for the Step meta for the variant:

```
export interface WpcomStep extends Step {
	meta: {
		heading: string | null;
		description: {
			desktop: string | null;
			mobile: string | null;
		};
		imgSrc?: {
			desktop?: {
				src: string;
				type: string;
			};
			mobile?: {
				src: string;
				type: string;
			};
		};
	};
}
```

Not final, but we do structure things as `{ mobile | desktop }` a lot, so may as well do it for everything? Feels almost like two separate configurations might work 😵

Reply:
> Feels almost like two separate configurations might work 😵

For example, we could have something like `config.step` and `config.stepDesktop` variables (or `config.step.{ default | desktop }`), both having the same structure:

```
interface Step {
    referenceElement?: string;
    options?: {
        classNames?: string[];
    };
    meta: {
        [ key: string ]: unknown;
    };
};

interface WpcomStep extends Step {
    meta: {
        heading?: string;
        description?: string;
        imgSrc?: {
            src: string;
            type: string;
        };
    };
}
```

if both are provided, then Tour Kit picks the right one for each case! I think I'm liking this! will see if can be applicable to the whole config 😆

Reply:
I just have the feeling that for a generic customer it'd be easier to just say: 
```css
options: {
  className: {
	  desktop: 'is-with-extra-padding',
	  mobile: null, //in this case no class applied to the step when in mobile
  },
},
```
rather than work with `is-mobile` and `is-desktop` modifiers. And as you said, that mimics how we structured the rest of the configuration 🤔

Reply:
We could ship this PR with the CSS I tried, as the main purpose is to add the new slide, and then try the new approach in a follow-up and see if we like it 😃

Reply:
Oh didn't saw your last comment, that's not bad at all! I'd explore it in another PR though 😄

Reply:
> if both are provided, then Tour Kit picks the right one for each case
I like this a lot to be honest, it seems just a lot easier for the user to use that in some way instead of working with CSS classes. It also should be easier to explain

Reply:
> I just have the feeling that for a generic customer it'd be easier to just say:

Ok, let's do that then and get rid of `is-desktop` / `is-mobile`. Let's handle it in another PR if you prefer (mind creating an issue? -> something specific like "Deprecate is-mobile/is-desktop modifiers in favor of granular className definitions".

> Oh didn't saw your last comment, that's not bad at all! I'd explore it in another PR though

Definitely not something for here. 😆  I'm exploring on my end as I'm revising the types with the variant extraction I think it will likely come with some variation of that. There's just too much in the way of `desktop | mobile` everywhere.

Reply:
Do you think we can ship this PR?